services/simcore/docker-compose.yml

```diff
@@ -268,6 +331,14 @@ services:
       placement:
         constraints:
           - node.labels.simcore==true
+      resources:
+        reservations:
+          cpus: "0.1"
+          memory: "256M"
+        limits:
+          cpus: "1"
```

Comment:
This is one of my nightmare services.
Q: can we have an alert if the service hits this CPU cap? could we diagnose somehow that it requires an increase of CPU?
I would like to have something similar for all of them actually. How could we achieve this?

Reply:
Let's make discuss it outside this PR